packages/neutrino/src/devServer.js

```diff
@@ -1,11 +0,0 @@
-const merge = require('deepmerge');
-const Future = require('fluture');
-const { serve, validate } = require('./webpack');
-
-// devServer :: (Object config -> Object api) -> Future () Function
-const devServer = (config, { options }) => Future
-  .of(merge({ devServer: { host: 'localhost', port: 5000, noInfo: !options.debug } }, config))
```

Comment:
Does the `noInfo: !options.debug` need to move to `packages/dev-server/index.js` now? Or are we leaving it to the user to pass the options to webpack-serve? 

The latter seems to make sense, I was just wondering if with the current settings things are going to get verbose. I think after this PR we'll probably want to revisit our customisation of `stats` entirely, so perhaps this can wait until then?

Reply:
Yeah let's follow up with stats changes.